### PR TITLE
Add Bvpy

### PR DESCRIPTION
**Pull request by floriangc**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by floriangc** [comment]
Hello @conda-forge/staged-recipes, we have a problem with the windows job that we dont want to trigger.
We addded `skip: true  # [win]` in the build section of the meta.yaml but it seems to have no effect ...
Is this normal ?

**Comment by floriangc** [comment]
Hello @conda-forge/staged-recipes, we want to disable the win jobs.
We added the line `skip: true  # [win]` in the build section of the meta.yaml but this seems not to work ..
Is this normal ?

File: recipes/bvpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  version: {{ version }}
+
+source:
+  url: https://gitlab.inria.fr/mosaic/bvpy/-/archive/master/bvpy-master.tar.gz
+  sha256: defe4cdba874a8a33a7d6424424d8de76bf3cbcef70b87cae681d31bbdbdf28d
+
+build:
+  skip: true  # [win or py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install {{ SRC_DIR }} -vv"
```
Comment (isuruf) [inline recipes/bvpy/meta.yaml:15]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/bvpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{% set name = "bvpy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.inria.fr/mosaic/bvpy/-/archive/master/bvpy-master.tar.gz
```
Comment (isuruf) [inline recipes/bvpy/meta.yaml:9]: We don't package `master` as `1.0.0`

Comment (floriangc) [reply recipes/bvpy/meta.yaml:9]: Ok, can I use instead the commit associated to the tag `1.0.0.` (https://gitlab.inria.fr/mosaic/bvpy/-/archive/1.0.0/bvpy-1.0.0.tar.gz) ?
Thank you for your time.

**Comment by floriangc** [comment]
@oalii, @gcerutti, @jlegrand62 do you confirm that you are willing to maintain this recipe ?

**Comment by oalii** [comment]
I confirm that I will maintain this recipe.

**Comment by gcerutti** [comment]
yes i will

**Comment by jlegrand62** [comment]
I confirm that I am willing to maintain this recipe.

**Comment by floriangc** [comment]
@conda-forge-admin, please ping team

**Pull request merged**